pom.xml

```diff
@@ -1129,10 +1129,31 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>com.linkedin.coral</groupId>
+                <artifactId>coral-hive</artifactId>
+                <version>${dep.coral.version}</version>
+            </dependency>
+
             <dependency>
                 <groupId>com.linkedin.coral</groupId>
                 <artifactId>coral-trino</artifactId>
                 <version>${dep.coral.version}</version>
+                <exclusions>
+                    <!--
+                          The trino-parser library is marked as provided dependency for
+                          coral-trino-parser.
+                          The trino-parser library is being used internally within
+                          coral-trino library for parsing Trino statements.
+                          However, this functionality is irrelevant in the context of
+                          using coral for Hive view translations within Trino and can be
+                          therefore safely excluded from the dependencies of this library.
+                    -->
```

Comment:
You explain why it feels safe-ish to do it, but we should also explain why we do it.
```
Dependency is excluded because trino-parser does not guarantee API or behavioral
backwards compatibility. We cannot provide version depended by Coral here, since
the current version is used in tests. To avoid breaking in weird way, or silent correctness
issues, the dependency is excluded, so that it is clear when someone tries to use the
non-backwards compatible classes.
```

Reply:
I do not think the exclusion is needed anyways, but I will you decide if we should keep it.

Reply:
see https://github.com/trinodb/trino/pull/11159#discussion_r816733470